README.md

```diff
@@ -1,35 +1,63 @@
-FreeIPA Ansible collection
+FreeIPA Ansible Collection
 ==========================
 
-This repository contains [Ansible](https://www.ansible.com/) roles and playbooks to install and uninstall [FreeIPA](https://www.freeipa.org/) `servers`, `replicas` and `clients`. Also modules for group, host, topology and user management.
-
-**Note**: The ansible playbooks and roles require a configured ansible environment where the ansible nodes are reachable and are properly set up to have an IP address and a working package manager.
-
-Features
---------
-* Server, replica and client deployment
-* Cluster deployments: Server, replicas and clients in one playbook
-* One-time-password (OTP) support for client installation
-* Repair mode for clients
-* Modules for dns forwarder management
-* Modules for dns record management
-* Modules for dns zone management
-* Modules for group management
-* Modules for hbacrule management
-* Modules for hbacsvc management
-* Modules for hbacsvcgroup management
-* Modules for host management
-* Modules for hostgroup management
-* Modules for pwpolicy management
-* Modules for role management
-* Modules for service management
-* Modules for sudocmd management
-* Modules for sudocmdgroup management
-* Modules for sudorule management
-* Modules for topology management
-* Modules fot trust management
-* Modules for user management
-* Modules for vault management
+The FreeIPA Ansible Collection provides [Ansible](https://www.ansible.com/) roles and playbooks to install and uninstall [FreeIPA](https://www.freeipa.org/) `servers`, `replicas` and `clients`. It also provides modules and playbooks to manage FreeIPA installations.
+
+> **Note**: A configured Ansible environment where the Ansible nodes are reachable and are properly set up is required for both roles and modules, and it might be required a properly configure IP address. The Ansible roles require a working package manager.
+
+
+Roles and Modules
+=================
+
+These are the roles and modules currently available. If you want to write a new module please read [writing a new module](plugins/modules/README.md).
+
+
+Roles
+-----
+
+Roles for server, replica and client deployment, with support for cluster deployments in a single playbook. Client role supports one-time-password (OTP) for installation, and repair mode.
+
+* [Server](docs/roles/README-ipaserver.md)
```

Comment:
As we have descriptions for the modules, we should also have descriptions for the roles.

Reply:
fixed.